setup.py

```diff
@@ -14,7 +14,7 @@
     url='https://github.com/DemocracyClub/dc_signup_form',
     install_requires=[
         'requests',
-        'Django >=1.10,<2.3',
+        'Django==3.2.0',
```

Comment:
This change means that it will only work with Django 3.2.0. We want a range here, starting at the lowest Django version we support (the first version to introduce `re_path` I expect?), and ending at `<3.3` — if we use less than 3.3 then we get all minor version bumps after 3.2.0 and before 3.3.0.

You should also look at the [CI matrix](https://github.com/DemocracyClub/dc_signup_form/blob/master/.travis.yml#L9) and add/remove the versions we want to support there.

Reply:
Looks like Django 2.0 was the first to add re_path: https://docs.djangoproject.com/en/2.0/ref/urls/